pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "goose-ai"
 description = "a programming agent that runs on your machine"
-version = "0.9.11"
+version = "0.9.12"
```

Comment:
did you mean to change it as part of this PR?

Reply:
Will undo, I had rolled it with another PR that was a feature addition.

Reply:
Updated, ready for re-review